user-guide/modules/ROOT/nav.adoc

```diff
@@ -1,5 +1,6 @@
 * xref:intro.adoc[Introduction to Boost]
 * xref:getting-started.adoc[]
+* xref:libraries-that-work-together.adoc[]
```

Comment:
I find it weird to take room in the level 0 hierarchy for such a specific type of content, especially since the intro.adoc still has a very poor and busy visual structure. I would imagine this kind of thing in an admonition in intro.adoc or something rather than as a level 0 section.

Reply:
@anarthal feedback?

Reply:
Could move it to the intro.

Reply:
We need to really dial down the length of these nav items.

Reply:
@alandefreitas agree. I'd expect a "this is what you can find in Boost" section instead. I've never seen such a section before anywhere in other libraries.